src/lib/y2storage/btrfs_raid_level.rb

```diff
@@ -27,15 +28,37 @@ module Y2Storage
   #
   # This is a wrapper for the Storage::BtrfsRaidLevel enum
   class BtrfsRaidLevel
+    include Yast::I18n
+    extend Yast::I18n
+
     include StorageEnumWrapper
 
     wrap_enum "BtrfsRaidLevel"
 
-    # Returns human readable representation of enum which is already translated
+    TRANSLATIONS = {
+      default: N_("Default"),
+      single:  N_("SINGLE"),
+      dup:     N_("DUP"),
+      raid0:   N_("RAID0"),
+      raid1:   N_("RAID1"),
+      raid5:   N_("RAID5"),
+      raid6:   N_("RAID6"),
+      raid10:  N_("RAID10")
+    }
+
+    private_constant :TRANSLATIONS
+
+    # Returns human readable representation of enum which is already translated.
+    #
+    # @raise [RuntimeError] when called on enum value for which translation is not defined yet.
     #
     # @return [String]
     def to_human_string
-      Storage.btrfs_raid_level_name(to_storage_value)
+      textdomain "storage"
+
+      value = TRANSLATIONS[to_sym] or raise "Unhandled Btrfs RAID level value '#{inspect}'"
```

Comment:
I kind of fear that `raise`. Do we have a unit test or some other mechanism to detect unhandled RAID levels before they hit the user?

Reply:
No, we don't. And the same for other things like MdParity.

Reply:
I have unit test to ensure everything is translated, and eureka! we have no translation for "UNKNOWN" RAID level :P

Reply:
Will you create a separate pull request for that?

Reply:
I see the fix is part of this PR already. Never mind.